cluster-operator/src/test/java/io/strimzi/operator/cluster/operator/assembly/KafkaBridgeAssemblyOperatorTest.java

```diff
@@ -66,8 +68,13 @@
 @RunWith(VertxUnitRunner.class)
 public class KafkaBridgeAssemblyOperatorTest {
 
-    private static final KafkaVersion.Lookup VERSIONS = new KafkaVersion.Lookup(Collections.emptyMap(),
-            Collections.emptyMap(), Collections.emptyMap(), Collections.emptyMap());
+    private static final KafkaVersion.Lookup VERSIONS = new KafkaVersion.Lookup(
+            new StringReader(KafkaVersionTestUtils.getKafkaVersionYaml()),
+            KafkaVersionTestUtils.getKafkaImageMap(),
+            KafkaVersionTestUtils.getKafkaConnectImageMap(),
+            KafkaVersionTestUtils.getKafkaConnectS2iImageMap(),
+            KafkaVersionTestUtils.getKafkaMirrorMakerImageMap()) { };
```

Comment:
If you put this inside some static getter in the `KafkaVersionTestUtils` then I think you can replace almost all the places in the tests where we have to instantiate a `KafkaVersion.Lookup` directly with a call to that method. I think the only exception will be in the `KafkaVersionsTest`.

Reply:
Yeah, yet again I was thinking the same as I replaced the tenth one in a row, there is a lesson here somewhere I think!